06_create_cluster.sh

```diff
@@ -22,4 +22,7 @@ for name in dnsmasq ironic-inspector ; do
     sudo podman ps | grep -w "$name$" && sudo podman stop $name
 done
 
+# Configure storage for the image registry
+oc --config ${KUBECONFIG} patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"storage":{"emptyDir":{}}}}'
```

Comment:
@ardaguclu was testing this and I think we discovered we need to change the operator state to Managed instead of Removed, too.

https://docs.openshift.com/container-platform/4.3/registry/configuring_registry_storage/configuring-registry-storage-baremetal.html#registry-change-management-state_configuring-registry-storage-baremetal